example/CalibratedTransform.yml

```diff
@@ -1,11 +1,12 @@
 %YAML:1.0
+---
```

Comment:
just curious, what caused the dashes to appear? no changes needed if it still conforms to yaml specs

Reply:
Honestly, I don't really know. YAML uses three dashes (“---”) to separate directives from document content. I know I had to refactor the file a bit before being able to load in it python without the three dashes but I don t remember changing anything to get there automatically set now..